app/_posts/blog/2016-05-29-2016-retrospective-planning-for-2017.md

```diff
@@ -0,0 +1,76 @@
+
+We also know that most of you didn't get too excited with the talks we had... We hear you loud and clear and we'll make sure to have an even more strict vetting process for next year's speakers. We will leverage our, now even bigger, network of communities throughout Greece, speakers from the two previous DEVits and pick up every stone we can turn so we can book and verify the best and most challenging speakers for DEVit 2017.
+
+Other than those 2 major fails we have made over 50 bullet points on things that we should be improving, tweaking and fixing. You see, DEVit is a community organized event, everything that you see about DEVit comes out of our free time. However this community is not comprised by hobbyists, we are professionals with decades of experience in project management, web development, design, marketing and all the arts, crafts and sciences that can make this work. Combine that with our excess amounts of OCD and perfectionism and you can be rest assured that you are up for a big treat for 2017.
+
+## Looking ahead, 2017 here we come!
+
+One of the problems of DEVit 2016 was that we started planning too late, we only really got started in January 2016, barely 5 months before the event. That amount of time was ridiculously small considering everything that happens for DEVit happens at our free time... That will no longer be the case...
+
+We have decided on the dates of DEVit for the next three years and have started planning for all of them from now, right the next day when DEVit 2016 finished.
+
+DEVit 2015 was an experiment, we didn't have any historical data, there was no other technological event of that scale organized anywhere in Greece, we had never organized such a big scale event before... We came through...
+
+DEVit 2016 was verification, we made sure that the numbers observed in 2015 were not a fluk and we could reliably verify them, attendees, sponsors, ticket price, venue, catering, all the major and minor parts that comprise a conference of that scale.
+
+DEVit 2017 will be all about scaling! Oh boy what do we have in store for you...
+
+### Two day conference
+
+Oh yes, we're becoming a two day conference, holding the main event with 2 tracks on Saturday 20th of May and a full day of workshops on Sunday the 21st. We were amazed by your interest in workshops, challenging workshops, and how much you valued them. So we're dedicating a full day for workshops in a multi-stage environment, hosting everything from full-day workshops down to 3, 2, or 1 hour sessions; whatever and as much time the presenter will need to convey their experience to you.
+
+A two day conference will require a conference dedicated hotel as well, so we've already gotten started to book a whole hotel for the days of the conference so all you coming from out of town can stay along with your peers and speakers in where the DEVit's heart will beat.
```

Comment:
we've already gotten started to book a whole hotel -> we started making the arrangements to fully book a hotel
so all you coming -> so all of you coming
in where -> where